docs/root/configuration/http/http_conn_man/headers.rst

```diff
@@ -587,7 +587,55 @@ Supported variable names are:
       <config_http_conn_man_headers_x-forwarded-for>`.
 
 %DOWNSTREAM_REMOTE_ADDRESS_WITHOUT_PORT%
-    Same as **%DOWNSTREAM_REMOTE_ADDRESS%** excluding port if the address is an IP address.
+    Remote address of the downstream connection. If the address is an IP address the output does
+    *not* include port.
```

Comment:
Given the name, why isn't this always without port?

Reply:
I copied the exact text that you commited 4 years ago, so that it would be consistent between `access_log/usage.rst` & `http_conn_man/headers.rst`
Here's how it is today in the `main` branch, exact same wording on the exact same variable, just in the other file.
https://github.com/envoyproxy/envoy/blame/main/docs/root/configuration/observability/access_log/usage.rst#L437-L443

Here's that commit:
https://github.com/envoyproxy/envoy/commit/1fdb6386c4bb42748530d7a9bf58ded644d77749#diff-d2fb733aedcdf06d58d1acff758025b22884906a7f2f071aaf3602dee46252efR128-R136

Since you committed that original version, why did you choose that wording, and is not still accurate? I'm happy to change both of them, but I stuck with the existing language rather than change both places.

I feel a refactor to only have one formatter would be beneficial in the long term, but I don't think this PR is the place.

Reply:
I didn't write the original, that PR you reference is a file migration across repositories (from data-plane-api to main repo). I think the wording is just confusing. I would change it to this:

"Remote address of the downstream connection without any port component. IP addresses are the only address type with a port component."

Reply:
Thanks, I've applied that new wording consistently in commit 1f6c7efcd1fa9f6474b4396fc3c8e018335ff9cb on this PR.